components/kyma-environment-broker/internal/process/update/initialisation_step.go

```diff
@@ -100,9 +127,18 @@ func (s *InitialisationStep) Run(operation internal.UpdatingOperation, log logru
 	return s.initializeUpgradeShootRequest(operation, log)
 }
 
+func (s *InitialisationStep) getRuntimeIdFromProvisioningOp(operation *internal.UpdatingOperation, log logrus.FieldLogger) error {
```

Comment:
log not used

Reply:
oversight from extracting method using IDE, thanks for spotting it, will be removed